### PR TITLE
Try to fix Travis CI

### PR DESCRIPTION
**Pull request by mammuth**

### CODE REVIEW

**Comment by mathiasquintero** [comment]
@mammuth:
```
[!] Could not find option 'output_files' in the list of available options: workspace, project, device, devices, scheme, clean, code_coverage, address_sanitizer, skip_build, output_directory, output_style, output_types, buildlog_path, formatter, derived_data_path, result_bundle, sdk, open_report, configuration, destination, xcargs, xcconfig, slack_url, slack_channel, slack_message, skip_slack, slack_only_on_failure, use_clang_report_name, custom_report_file_name, fail_build
```

I'm gonna guess they renamed the parameter. Is it `output_directory` maybe?

**Comment by mammuth** [comment]
Wasn't it you changing exactly that recently?

**Comment by rodalfus** [comment]
No, that was me. I guess with the new osx_image, they downgraded(?) Fastlane again. I’m pretty sure it’s actually output_file or something like that. There should be a relatively recent closed PR with the change, though.

**Comment by mathiasquintero** [comment]
@mammuth probably this change: https://github.com/TCA-Team/iOS/pull/143/commits/46db8839a8d5443522ac47b64498b4c79ce45f1f#diff-56c5021ef06ac6cea807ae53269b6f4fL48

**Comment by rodalfus** [comment]
@mathiasquintero Exactly, that should be it. We had to change the Travis `osx_image` from 8.2 to 8.3 and for some reason, that updated Fastlane as well, deprecating the previously used options.

**Comment by mammuth** [comment]
Are we sure that the Xcode8.3 image supports Swift4? Travis also has a 9 and 9.1 image

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: objective-c  # [sic]. Travis uses objective-c just to determine it needs to use the xcode cli
-osx_image: xcode8.3
+osx_image: xcode9.1
```
Comment (mathiasquintero) [inline .travis.yml]: Why 9.1. Isn't it still beta?

Comment (mammuth) [reply .travis.yml]: Travis [doesn't state anything](https://docs.travis-ci.com/user/reference/osx/#OS-X-Version) the like

**Comment by mammuth** [comment]
Wohooo, fixed! 🎉 🍻

**Comment by mathiasquintero** [comment]
Merged!!!!

**Pull request merged**